lib/src/ui/game/archived_game_screen.dart

```diff
@@ -198,31 +198,45 @@ class _BottomBar extends ConsumerWidget {
                   : null,
               icon: const Icon(LichessIcons.fast_backward, size: 20),
             ),
-            BottomBarIconButton(
-              key: const ValueKey('cursor-back'),
-              // TODO add translation
-              semanticsLabel: 'Backward',
-              onPressed: canGoBackward
-                  ? () {
-                      ref
-                          .read(gameCursorProvider(gameData.id).notifier)
-                          .cursorBackward();
-                    }
+            LongPressButton(
+              callback: canGoBackward
+                  ? () => ref
+                      .read(gameCursorProvider(gameData.id).notifier)
+                      .cursorBackward()
                   : null,
-              icon: const Icon(LichessIcons.step_backward, size: 20),
+              child: BottomBarIconButton(
+                key: const ValueKey('cursor-back'),
+                // TODO add translation
+                semanticsLabel: 'Backward',
+                showTooltip: false,
+                onPressed: canGoBackward
```

Comment:
the `onPressed` callback should be a no op to explicitly show the real callback is the one of the parent widget.

Reply:
I don't think it can be a no op as the `onTap` is delegate to the child widget so it will do nothing on tap. It can also not be null as it won't be highlighted. I have changed the name of the callback in the `RepeatButton` to be `longPressCallback` and added a bool `handleOnTap` to be more clear. Added doc as well.

Reply:
My understanding was that `HitTestBehavior.opaque` prevented all touch events to reach the child, and the `RepeatButton` would handle the `onTap` itself.

Reply:
I think the documentation is a bit weird. Here is an explanation https://github.com/flutter/flutter/issues/18450#issuecomment-397372078

Reply:
ok then I suppose we can declare the callback as a private method to avoid duplicate the code